.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '1.20'
+      - uses: actions/checkout@v3
+      - uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.52.2
+          args: --timeout=5m
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '1.20'
+      - uses: actions/checkout@v3
+      - run: go test -race ./...
+
+  build:
+    strategy:
+      matrix:
+        go-version: [ ^1.20 ] # ^1.19
+        os: [ ubuntu-latest ] # macos-latest, windows-latest
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.go-version }}
+      - uses: actions/checkout@v3
+      - run: go build ./cmd/chat-service
```

Comment:
newline

https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3

https://www.google.com/search?q=why+newline+at+end+of+file

<img width="383" alt="Screenshot 2023-04-16 at 08 33 42" src="https://user-images.githubusercontent.com/17127404/232273112-8b1103b8-54d5-476a-bcdd-2d2d9ce527ea.png">